CaesarCipher.py

```diff
@@ -0,0 +1,118 @@
+'''In this exercise you will implement the class CaesarCipher, which represents
```

Comment:
[Please read this comment as the last one]

I think it is better to add a new function called shift_string(s, shift) which does exactly what encrypt is doing, but takes the shift as a parameter.

In this way we can rewrite:
```
def self.encrypt(self, plaintext):
    return self.shift_string(s, self.key)
```
and
```
def self.decrypt(self, plaintext):
    return self.shift_string(s, -self.key)
```

By writing the code this way we avoid an unnecessary duplication of the code, which makes it less bug-prone.